monitoring/ping.go

```diff
@@ -291,3 +284,16 @@ func (c *pingChecker) calculateRTT(serfClient agent.SerfClient, self, node serf.
 		otherNodeCoord.Vec, otherNodeCoord.Error, otherNodeCoord.Height, otherNodeCoord.Adjustment)
 	return latency, nil
 }
+
+// failureProbe constructs a new probe that represents a failed ping check
+// against the specified node.
+func (c *pingChecker) failureProbe(node string, latency int64) *pb.Probe {
+	return &pb.Probe{
+		Checker: c.Name(),
+		Detail: fmt.Sprintf("ping between %s and %s is higher than the allowed threshold of %dms",
+			c.self.Name, node, latencyThreshold.Milliseconds()),
```

Comment:
I was wondering if you also wanted to sample down to ms in the log i.e. [here](https://github.com/gravitational/satellite/pull/216/files#diff-b19d95e07445b85e4c38f5292c7183e4R204)?

Reply:
Logs time format updated in a9c489f